### ISSUE TITLE
Reworking levels

### ISSUE THREAD

**Issue by SirGiraffey** [fishfolk/jumpy#697]
Reworking and redesigning all levels to use the newer tilemaps, critters, weapons etc. using the new editor.

Here's the first 3!

[Level 1.txt](https://github.com/fishfolk/jumpy/files/10936391/Level.1.txt)
[Level 2.txt](https://github.com/fishfolk/jumpy/files/10936392/Level.2.txt)
[Level 3.txt](https://github.com/fishfolk/jumpy/files/10936393/Level.3.txt)

I'm attaching them as txt files, let me know if that suits you

**Comment by zicklag** [comment]
Awesome! That'll work fine for me. 👍

**Comment by zicklag** [comment]
@SirGiraffey I was able to re-order the layers in Level 3 so you get this nice foreground action where some times can be on top of the players/items/critters:

![image](https://user-images.githubusercontent.com/25393315/224177686-56049b80-a153-46b5-a415-acdf08aacf7a.png)

Just wanted to make sure you wanted it like that.

Also, I kind of liked the old level 2. Do you think we could take your new level 2 and add it as a new map so we can keep the old one, too?

**Comment by SirGiraffey** [comment]
Hey, I remade a few next levels. I love the foreground effect you did with the nets.
And I'll remake the 2. level again then, without changing it too much :)

[Level 4.txt](https://github.com/fishfolk/jumpy/files/10962622/Level.4.txt)
[Level 5.txt](https://github.com/fishfolk/jumpy/files/10962623/Level.5.txt)
[Level 6.txt](https://github.com/fishfolk/jumpy/files/10962624/Level.6.txt)
[Level 7.txt](https://github.com/fishfolk/jumpy/files/10962619/Level.7.txt)
[Level 8.txt](https://github.com/fishfolk/jumpy/files/10962620/Level.8.txt)
[Level 9.txt](https://github.com/fishfolk/jumpy/files/10962621/Level.9.txt)

**Comment by zicklag** [comment]
Oh sweet! I'm just getting ready to sit down with my bros and do some testing, so we'll try out your new levels, too!

**Comment by SirGiraffey** [comment]
Sending 2 more levels!

[Level 10.txt](https://github.com/fishfolk/jumpy/files/11012399/Level.10.txt)
[Level 11.txt](https://github.com/fishfolk/jumpy/files/11012401/Level.11.txt)

By the way, do you have any feedback from testing the other levels?

**Comment by zicklag** [comment]
The biggest feedback was just artistically maybe it'd still be good to incorporate the old tileset in some of the maps, since the old tileset still looked good, but that's also just style preference.

As far as gameplay me and my bros. realized that it's a little hard to know exactly how it's going to play in the final version where you're not just re-spawning constantly, and it actually organizes into rounds. So no solid feedback there until we can test with round system, since I think that could change a lot of the strategy.

**Comment by SirGiraffey** [comment]
Here's the last 2 levels for now.
[Level 12.txt](https://github.com/fishfolk/jumpy/files/11023767/Level.12.txt)
[Level 13.txt](https://github.com/fishfolk/jumpy/files/11023768/Level.13.txt)

Yeah I agree with the levels lacking artistically, but I tried to do the most of the assets. The problem with the old tileset is that its misalligned compared to the new ones. Once the tiles are synchronised switching the tilesets of any levels won't be a problem at all.
I feel like the old tileset could use a touch from the artists, cause I think it fits the previous iteration of assets, not the current style direction. We're also lacking many decorations/backgrounds, so all levels look very similarly despite my best efforts.

I've got a small list of things that I felt could be changed within the editor but I'll think it through tomorrow and post it later :)
